### ISSUE TITLE
New Rule:  Suggest using Node.js built-in modules over third-party packages

### ISSUE THREAD

**Issue by aladdin-add** [eslint-community/eslint-plugin-n#563]
### Rule details

Node.js now ships many APIs that replace popular third-party packages (e.g., util.styleText for terminal coloring, global fetch, crypto.randomUUID). Yet there is no rule in eslint-plugin-n to warn when a package can be replaced by a built-in.

The most prominent example: chalk → util.styleText. ESLint itself migrated its stylish formatter from chalk to util.styleText in v10.0.0.

### What type of rule is this?

Suggests an alternate way of doing something

### Example code

```js
// ❌ Incorrect
import chalk from 'chalk';
chalk.red('error');
chalk.bold.red('error');

// ✅ Correct
import { styleText } from 'node:util';
styleText('red', 'error');
styleText(['bold', 'red'], 'error');
```

### Participation

- [ ] I am willing to submit a pull request to implement this rule.

## Prior Art

* @friendsoftheweb/eslint-plugin has a ban-chalk rule with auto‑fix for chalk→styleText conversion.

* This rule would integrate the same idea into eslint-plugin-n’s existing infrastructure (target version detection, fixability).

**Comment by MichaelDeBoey** [comment]
We can probably rely on [`nativeReplacements`](https://github.com/e18e/module-replacements#native-replacements-nativereplacements-nativejson) from @e18e's [`module-replacements`](https://npmx.dev/module-replacements) and should probably also have a look at what @43081j did for [`@e18e/eslint-plugin`](https://npmx.dev/@e18e/eslint-plugin)'s [`ban-dependencies` rule](https://github.com/e18e/eslint-plugin/blob/main/src/rules/ban-dependencies.ts)

**Comment by voxpelli** [comment]
I think we should leave this type of work for the e18e plugin / community to do

Let's keep this plugin focused on node.js itself, not have it care about npm modules, e18e already does that
